cloner/cloner.go

```diff
@@ -35,6 +35,8 @@ func NewDiskCloner(baseDir string) *DiskCloner {
 
 			return diskCloner
 		}
+
+		log.Infoln("Clone directory provided does not exist. Will clone in default cache directory.")
```

Comment:
I think it can be a warning, it should be printed even without `-v/--verbose` and in the usage of `--clone-dir`, we use the term "cloning location".
```suggestion
		log.Warnln("Cloning location provided does not exist. Will clone in default cache directory.")
```